[TextAnalytics] Remaining Extractive Text Summarization implementation work (#22841)
Azure/azure-sdk-for-net: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

## 5.2.0-beta.1 (Unreleased)

> 

Reviewer: I guess we missed it in the previous PR, we need to add the service bump version

Author: Updated.

Author: Noticed my push didn't work and the changelog was not updated. Adding the change to the next PR.

### Features Added

> - Added property `ExtractSummaryActions` to `TextAnalyticsActions` to support the new 'extractive text summarization' API. This action can be used to extract relevant sentences from an input document.

Reviewer 2: I think we need to be careful about describing this action because it applies a well-known NLP analysis and "relevant sentences" does not sound like the best description. Here is one suggestion:
```suggestion
- Added property `ExtractSummaryActions` to `TextAnalyticsActions` to support the new 'extractive text summarization' API. This action can be used to get a summary for the input document by extracting the most relevant sentences.
```
